arch/xtensa/src/common/xtensa_attr.h

```diff
@@ -63,4 +69,10 @@
 
 #define RTC_RODATA_ATTR __attribute__((section(".rtc.rodata")))
 
+/* Forces bss variable into external memory. */
```

Comment:
what kind of data needs to be forced to be in spiram?

if its purpose is to save internal memory, i guess it's more useful to make it "allow to be in spiram", rather than force. ie. make `EXT_RAM_ATTR` no-op unless `CONFIG_XTENSA_EXTMEM_BSS` is enabled.

Reply:
The data that usually gets in SPIRAM are large bss data. Yes, the purpose is to save some internal memory.

I made EXT_RAM_ATTR a no-op when CONFIG_XTENSA_EXTMEM_BSS is not enabled and changed the "force" to "allow", although, I find that when we are using the attribute we are actually forcing the variable to reside in that section.